script/tracking.js

```diff
@@ -0,0 +1,179 @@
+var clicks = 0;
+var totalKeys = 0;
+var startMs;
+var totalHours = 0;
+var totalMinutes = 0;
+var totalSeconds = 0;
+
+var buttonClicks = 0;
+var userK = 0;
+var userC = 0;
+var emailK = 0;
+var emailC = 0;
+var pwK = 0;
+var pwC = 0;
+var fnameK = 0;
+var fnameC = 0;
+var lnameK = 0;
+var lnameC = 0;
+var addressK = 0;
+var addressC = 0;
+var countryK = 0;
+var countryC = 0;
+var zipcodeK = 0;
+var zipcodeC = 0;
+var sexK = 0;
+var sexC = 0;
+var languageK = 0;
+var languageC = 0;
+var aboutK = 0;
+var aboutC = 0;
+
+//track the mouse clicks
+document.onclick = function(){
+    clicks++;
+    document.getElementById("track-clicks").innerHTML="Number of mouse clicks: " + clicks;
```

Comment:
it seems that changing the text contents (of all elements) can happen all together once when submit is clicked? (i.e. in showDiv function)